lms/djangoapps/teams/static/teams/js/spec_helpers/team_spec_helpers.js

```diff
@@ -29,13 +29,16 @@ define([
 
     var createMockTeamData = function (startIndex, stopIndex) {
         return _.map(_.range(startIndex, stopIndex + 1), function (i) {
+            var id = "id" + i;
             return {
                 name: "team " + i,
-                id: "id " + i,
+                id: id,
                 language: testLanguages[i%4][0],
                 country: testCountries[i%4][0],
                 membership: [],
-                last_activity_at: ''
+                last_activity_at: '',
+                topic_id: 'topic_id' + i,
```

Comment:
It was moved out because it is referenced 2 places. This isn't the case with topic_id.